### DISCUSSION TITLE
Some problems about SCSS file

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#706] [Q&A] [answered]
When integrating itwins into the customer's platform, SCSS is not supported. SCSS is used in itwins. I see an extension webpack tools module. Is this helpful?

**Comment by calebmshafer** [comment, marked as answer]
@shenshengyi, Extension Webpack tools is used to build runtime Extensions and not for use on an application.  Are you using `@bentley/react-scripts` to compile your web app or are you currently handling it a different way?

**Reply by shenshengyi** [reply]
It's settled. Thank you.
